src/patternfly/upgrade-examples/PopoverUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,13 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
```

Comment:
add in comparison between `arrow` and `pf-c-popover__arrow`

Reply:
Arrow comparisons has been added.